hpat/datatypes/hpat_pandas_series_functions.py

```diff
@@ -198,6 +198,79 @@ def hpat_pandas_series_values_impl(self):
     return hpat_pandas_series_values_impl
 
 
+@overload_method(SeriesType, 'var')
+def hapt_pandas_series_var(self, axis=None, skipna=None, level=None, ddof=1, numeric_only=None):
+    """
+    Pandas Series method :meth:`pandas.Series.var` implementation.
+
+    .. only:: developer
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_var
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_var_unboxing
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_var_str
+       Test: python -m hpat.runtests hpat.tests.test_series.TestSeries.test_series_var_unsupported_params
+
+    Parameters
+    ----------
+    self: :obj:`pandas.Series`
+        input series
+    axis: :obj:`int`, :obj:`str`
+        Axis along which the operation acts
+        0/None - row-wise operation
+        1      - column-wise operation
+        *unsupported*
+    skipna: :obj:`bool`
+        exclude NA/null values
+    level: :obj:`int`, :obj:`str`
+        If the axis is a MultiIndex (hierarchical),
+        count along a particular level, collapsing into a scalar
+        *unsupported*
+    ddof: :obj:`int`
+        Delta Degrees of Freedom.
+        The divisor used in calculations is N - ddof,
+        where N represents the number of elements.
+    numeric_only: :obj:`bool`
+        Include only float, int, boolean columns.
+        If None, will attempt to use everything, then use only numeric data.
+        Not implemented for Series.
+        *unsupported*
+
+    Returns
+    -------
+    :obj:`scalar`
+         returns :obj:`scalar`
```

Comment:
It can return scalar or Series

Reply:
I'm not sure we have to add `series` here, because `series` is returned in case of `level` specified, but currently we don't support parameter `level`. So `scalar` is always returned in our case. Do you still think I need to add `series` here?